### ISSUE TITLE
Set zoom causes view to shrink

### ISSUE THREAD

**Issue by srodenberg** [dexlex/DexMovingImageView#4]
If you start with a zoom of 2 in an xml, then change the zoom to 1 programmatically, the view shrinks. Shouldn't the view maintain its position and the zoom of the drawable in the view should change?

```
<it.dex.movingimageviewlib.DexMovingImageView
    android:id="@+id/image"
    android:layout_width="match_parent"
    android:layout_height="420dp"
    android:layout_alignParentTop="true"
    app:drawer="scale"
    app:zoom="2"
    app:angle="0"
    app:generator="base"
    app:evaluator="simple"
    tools:src="@drawable/hotdogs1400" />
```

That is the xml I have, I would like to scale this image so that it keeps its position but shrinks in size. Similar to a ken burns effect like shown here https://www.youtube.com/watch?v=nNTiyRDhlRY

**Comment by dexlex** [comment]
Good point. You can not do it for now because they are all asynchronous operations. You can create your generator to make the ken burns effect. Otherwise, You can wait for me, because I add this as a new feature for the next version.

Thank you
